probe-rs/src/probe/stlink/mod.rs

```diff
@@ -409,7 +410,9 @@ impl STLink {
     /// Returns Ok(()) otherwise.
     /// This can be called on any status returned from the attached target.
     fn check_status(status: &[u8]) -> Result<(), DebugProbeError> {
+        log::trace!("check_status({:?})", status);
         if status[0] != Status::JtagOk as u8 {
+            log::debug!("check_status failed: {:?}", status);
```

Comment:
The error happens quite a lot and also gets handled, so higher log levels like `warn` don't make sense here.